src/include/migraphx/op/scatter_add.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_SCATTER_ADD_HPP
+#define MIGRAPHX_GUARD_OPERATORS_SCATTER_ADD_HPP
+
+#include <array>
+#include <migraphx/check_shapes.hpp>
+#include <migraphx/stringutils.hpp>
+#include <migraphx/streamutils.hpp>
+#include <migraphx/shape_for_each.hpp>
+#include <migraphx/config.hpp>
+#include <migraphx/value.hpp>
+#include <migraphx/op/normalize_attribute.hpp>
+#include <cmath>
+#include <utility>
+#include <migraphx/op/scatter.hpp>
+
+// ScatterElement op. with "add" as the reduction attribute.
```

Comment:
This should say "scatter with an add reduction". MIGraphX doesn't have a ScatterElement op nor a reduction attribute.